source/network-security.txt

```diff
@@ -12,18 +12,137 @@ Network Security
    :depth: 1
    :class: onecol
 
-Intro statement
+By default, {+service+} secures your database deployments through the
+use of mandatory TLS/SSL connection encryption, and a VPC-first,
+minimum-access default policy. You can further configure these
+protections to meet your unique security needs and preferences.
 
-{+service+} Features and Best Practices for Network Security
-------------------------------------------------------------
+Use the foundational guidance on this page to plan the network
```

Comment:
I would remove foundational and instead say: "Use the basic recommendations on this page to plan the network ...."